ci/authn-k8s/dev/policies/policy.template.yml

```diff
@@ -198,6 +203,31 @@
   role: !group conjur/authn-k8s/minikube/clients
   member: !layer some-policy
 
+# This policy is for testing that in case a host doesn't have any annotations defined
+# then the "authentication-container-name" annotation is defaulted to "authenticator"
+# and the host is authenticated.
+# We have a separate policy for this host so we can use the id '{{CONJUR_AUTHN_K8S_TEST_NAMESPACE}}/*/*'
+# without duplication.
+- !policy
+  id: host-without-container-name
+  body:
+    - !layer
+
+    - &hosts
+      - !host
+        id: {{CONJUR_AUTHN_K8S_TEST_NAMESPACE}}/*/*
+        annotations:
+          kubernetes/authentication-container-name: authenticator
```

Comment:
This host shouldn't have the annotation defined, right?

Reply:
correct again